### ISSUE TITLE
How to fix CORS error on 3rd party API call in Nuxt 3?

### ISSUE THREAD

**Issue by matrixmandeep** [nuxt/nuxt#23538]
I am working on Nuxt 3 with useAsyncData hook to call API. When I call the 3rd party API post https://apitest.bankfeeds.com.au/v1/customer/data using useAsyncData hook, it returns the CORS erro

**Comment by huang-julien** [comment]
Hi :wave: can you provide a reproduction please ?

**Comment by matrixmandeep** [comment]
I'm trying to integrate **API** from marketo which only runs on **server side** and will not run on browser. 

**Check below:**
https://developers.marketo.com/rest-api/endpoint-reference/lead-database-endpoint-reference/#!/Leads/associateLeadUsingPOST

/rest/v1/leads/{leadId}/associate.json, this particular API is returning me CORS issues.

**Comment by huang-julien** [comment]
Yes but we need to see a code reproduction such as something on a git repository or stackblitz

**Comment by matrixmandeep** [comment]
Hi @huang-julien this issue is fixed for me after upgrading my project to latest nuxt version. But can you please tell me that is there any way to get the cookies on server side.
